builder/qemu/builder.go

```diff
@@ -154,8 +154,16 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, error) {
 		if runtime.GOOS == "windows" {
 			b.config.Accelerator = "tcg"
 		} else {
-			b.config.Accelerator = "kvm"
+			if fp, err := os.Open("/dev/kvm"); err != nil {
```

Comment:
It looks like this changes the default behavior to `tcg` in a case where `kvm` is either not present or is in a different spot. I think what might happen here is someone would expect `kvm` but find `tcg` in their debug output and be confused that it says `tcg` is detected.

If we want to preserve the existing functionality as closely as possible I think we should do this instead:

```
b.config.Accelerator = "kvm"
if !Exists("/dev/kvm") && Exists("/dev/tcg") {
    b.config.Accelerator = "tcg"
}
```